### DISCUSSION TITLE
Dialog that halt daily build

### DISCUSSION THREAD

**Discussion by bero** [community/community#62641] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi, I am member of a team at company and we have some private repositories.
Every morning we do a scheduled build. Since a while back there is a dialog that stops the build like this.

![image](https://github.com/community/community/assets/138818/89e54eaa-5b31-46dc-87b5-61aefef4502b)

SSH is used for the authentication. Can also mention I am not an expert in git and github.
Just use it to commit my changes :)

I have tried to login with browser. It works but problem is that dialog appear on next build.
Would appreciate any help!

**Comment by QuocHuy03** [comment]
huydev #2
